web/modules/custom/collection/config/install/core.entity_form_display.rdf_entity.collection.default.yml

```diff
@@ -151,13 +146,6 @@ content:
     third_party_settings: {  }
     type: inline_entity_form_complex
     region: content
-  field_ar_creation_date:
-    weight: 20
-    settings:
-      behaviour: create
```

Comment:
Let's also make sure that the new `created` and `changed` fields are configured as hidden in this form display. This information is not actually visible in the UI, so we should not fire validate handlers for them.